migrations/2025-06-14-141408_person_votes/up.sql

```diff
@@ -0,0 +1,61 @@
+ALTER TABLE person_actions
+    ADD COLUMN voted_at timestamptz,
```

Comment:
The timestamp doesnt seem so useful, anyway there would be no space to show it in UIs.